grammars/csharp.tmLanguage.cson

```diff
@@ -2761,29 +2782,60 @@ repository:
         include: "#comment"
       }
     ]
-  "checked-unchecked-statement":
```

Comment:
this removed the definition of `checked-unchecked-statement` however it is still referenced: https://github.com/dotnet/csharp-tmLanguage/blob/1933252dc3068f37d93140c31ad5dbd5753687b3/grammars/csharp.tmLanguage.cson#L226

this is breaking my more-strict tmlanguage parsing in https://github.com/asottile/babi-grammars:

```pytb
Traceback (most recent call last):
  File "/tmp/babi-grammars/bin/test-grammars", line 73, in <module>
    raise SystemExit(main())
                     ^^^^^^
  File "/tmp/babi-grammars/bin/test-grammars", line 47, in main
    compiler = grammars.compiler_for_file(filename, line)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 714, in compiler_for_file
    return self.compiler_for_scope(scope)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 689, in compiler_for_scope
    ret = self._compiled[scope] = Compiler(grammar, self)
                                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 528, in __init__
    root = self._compile_root(grammar)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 592, in _compile_root
    regs, rules = self._patterns(grammar, grammar.patterns)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 565, in _patterns_
    tmp_regs, tmp_rules = self._include(
                          ^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 547, in _include_
    return self._patterns(grammar, (repository[s[1:]],))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 571, in _patterns_
    tmp_regs, tmp_rules = self._patterns(grammar, rule.patterns)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 565, in _patterns_
    tmp_regs, tmp_rules = self._include(
                          ^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 547, in _include_
    return self._patterns(grammar, (repository[s[1:]],))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 571, in _patterns_
    tmp_regs, tmp_rules = self._patterns(grammar, rule.patterns)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 565, in _patterns_
    tmp_regs, tmp_rules = self._include(
                          ^^^^^^^^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/highlight.py", line 547, in _include_
    return self._patterns(grammar, (repository[s[1:]],))
                                    ~~~~~~~~~~^^^^^^^
  File "/tmp/babi-grammars/.tox/py312/lib/python3.12/site-packages/babi/fdict.py", line 45, in __getitem__
    raise KeyError(key)
KeyError: 'checked-unchecked-statement'
```

Reply:
Good catch. Opened https://github.com/dotnet/csharp-tmLanguage/pull/298 to remove this reference.